firmware_motion_board/src/motor.c

```diff
@@ -0,0 +1,83 @@
+#include "motor.h"
+#include <stdio.h>
+#include <stdbool.h>
+
+int flip_left = 1;
+int flip_right = 1;
+
+int motor_active = 0;

```

Comment:
```suggestion
static int flip_left = 1;
static int flip_right = 1;

static int motor_active = 0;

```
Ako je privatno polje onda dodati `static`

Reply:
Dodato static.